src/index.html

```diff
@@ -3,11 +3,330 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="shortcut icon"
+      type="image/vnd.microsoft.icon"
+      href="//static.bose.com/etc/designs/bose/consumer-products-2016/favicon.ico"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
```

Comment:
pay attention to mobile resolution.
Elements start setting off the desired display:
![mobile resolution](https://user-images.githubusercontent.com/39308717/163125896-0629a6cd-84ab-464c-bff4-7122a8623648.png)

Reply:
I think have tried to fix this one too. It wasn't an easy one for me. :)